otel-extension/src/main/java/io/scalac/mesmer/otelextension/akka/MesmerAkkaActorInstrumentationModule.java

```diff
@@ -81,6 +82,15 @@ public List<String> getAdditionalHelperClassNames() {
         "io.scalac.mesmer.otelextension.instrumentations.akka.actor.util.EnvelopeContext$",
         "io.scalac.mesmer.otelextension.instrumentations.akka.actor.impl.otel.ActorCellInstrumentationState",
         "akka.actor.ProxiedQueue",
-        "akka.actor.BoundedQueueProxy");
+        "akka.actor.BoundedQueueProxy",
+        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.extension.ActorLifecycleMonitorExtension",
```

Comment:
The otel muzzle plugin could make this simple, but I am not sure it could apply to SBT.

Reply:
Oh yeah! For sure. :) It's just temporary. I have plans to use the muzzle plugin but it's sadly not supported by sbt. I think we are going to end up migrating to gradle in some not-so-distant future as this is the tool that the otel opentelementry-java /opentelemetry-java-instrumentations projects use.